example: simplify orthographic controls

zarov opened the pull request:
Removed the kind of hacky controls of the orthographic example that I really didn't like and
replaced them by the PlanarControls provided by iTowns.

peppsac commented on examples/orthographic.js:
```diff
@@ -10,34 +10,18 @@ var extent = new itowns.Extent(
-    view.notifyChange(true);
-};
+// disable the rotation
+controls.handleRotation = function _() {};
```
I'd prefer an option in `PlanarControls` constructor to disable rotation, instead of overriding the function.
(and currently, clicking the middle button will still rotate the camera)

peppsac replied:
Also adding a zoom-speed option would be nice: the default zoom speed is slooooow :)

zarov replied:
I was hesitant to change the core, but I'm ok with adding an option (damn middle button, didn't see that one). I'm tweaking a speed too !

zarov commented:
Updated

peppsac commented on examples/orthographic.js:
```diff
@@ -10,34 +10,20 @@ var extent = new itowns.Extent(
+
+// eslint-disable-next-line no-new
+new itowns.PlanarControls(view, {
+    maxAltitude: 40000000,
```
Would you mind adding the documentation for these parameters?

zarov replied:
Do you mean here or in `PlanarControls` ?

peppsac replied:
In PlanarControls' jsdoc

elemoine replied:
Explaining the option values used here would make sense to me too. For example, while reading the example code I would like to understand why `enableRotation` is set to `false`.

zarov replied:
Doing both !

elemoine replied:
And on the `maxAltitude` value: why not making it bigger? As a user of the example I want to be able to zoom out, and the current `maxAltitude` value prevents it. I tried to change it to 16000000 but the map disappears when I zoom out.

zarov replied:
Concerning the `maxAltitude`, I had the same error as you, so I suggest to let it as it is.

peppsac replied:
Probably another issue with `camera.far`...
#674  would help fixing this by providing a min/max distance for everything we need to display => we could dynamically adjust near/far

elemoine commented:
I just added two comments, but overall I really like the idea of removing the custom controls that were used in this example.

zarov committed:
example: simplify orthographic controls

Remove the kind of hacky controls of the orthographic examples and
replace them by the PlanarControls provided by iTowns.

zarov committed:
docs: add doc for PlanarControls constructor

Clean the other doc in PlanarControls as well, but hide it from the
jsdoc generation.

zarov commented:
So I cleaned the documentation of `PlanarControls` and added an explanation of options. Note that only the construction and its options are generated in the final documentation, as I felt that a lambda user will have no interest in the others methods, but a more advanced user could still take a look at it by going into the code.

peppsac reviewed (approved):
LGTM, thanks
